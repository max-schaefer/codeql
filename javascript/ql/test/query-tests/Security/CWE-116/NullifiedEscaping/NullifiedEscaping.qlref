Security/CWE-116/NullifiedEscaping.ql
