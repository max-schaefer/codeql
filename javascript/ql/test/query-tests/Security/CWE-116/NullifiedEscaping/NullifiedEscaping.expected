nodes
| tst.js:2:3:2:51 | s |
| tst.js:2:7:2:30 | s.repla ... "\\\\\\\\") |
| tst.js:2:7:2:30 | s.repla ... "\\\\\\\\") |
| tst.js:2:7:2:51 | s.repla ...  '\\\\"') |
| tst.js:3:29:3:29 | s |
| tst.js:3:29:3:29 | s |
edges
| tst.js:2:3:2:51 | s | tst.js:3:29:3:29 | s |
| tst.js:2:3:2:51 | s | tst.js:3:29:3:29 | s |
| tst.js:2:7:2:30 | s.repla ... "\\\\\\\\") | tst.js:2:7:2:51 | s.repla ...  '\\\\"') |
| tst.js:2:7:2:30 | s.repla ... "\\\\\\\\") | tst.js:2:7:2:51 | s.repla ...  '\\\\"') |
| tst.js:2:7:2:51 | s.repla ...  '\\\\"') | tst.js:2:3:2:51 | s |
#select
| tst.js:3:10:3:30 | decodeU ... nent(s) | tst.js:2:7:2:30 | s.repla ... "\\\\\\\\") | tst.js:3:29:3:29 | s | This replacement nullifies escaping of '\\' $@.. | tst.js:2:7:2:51 | s.repla ...  '\\\\"') | here |
